Kernel/Bus/VirtIO/Console.cpp

```diff
@@ -64,9 +68,11 @@ UNMAP_AFTER_INIT bool Console::initialize()
     return true;
 }
 
-UNMAP_AFTER_INIT Console::Console(PCI::Address address)
+UNMAP_AFTER_INIT Console::Console(PCI::Address address, NonnullOwnPtr<Memory::RingBuffer>&& control_transmit_buffer, NonnullOwnPtr<Memory::RingBuffer>&& control_receive_buffer)
```

Comment:
There's no need to specify `&&` here, as we're not moving *from* the NonnullOwnPtr, were moving it. (Same in all the other ones)